addons/web_editor/static/src/components/upload_progress_toast/upload_service.js

```diff
@@ -106,7 +106,20 @@ export const uploadService = {
                 // limited by bandwidth.
                 for (const sortedFile of sortedFiles) {
                     const file = progressToast.files[sortedFile.progressToastId];
-                    const dataURL = await getDataURLFromFile(sortedFile);
+                    let dataURL;
+                    try {
+                        dataURL = await getDataURLFromFile(sortedFile);
+                    } catch {
+                        deleteFile(file.id);
+                        env.services.notification.add(
+                            _.str.sprintf(
+                                env._t('Could not load the file "%s". Please select a single file.'),
```

Comment:
The upload service can be used to upload multiple files. Therefore, the error message does not always give correct instructions to the user.

I would therefore either remove the second part of the error message either provide a relevant hint to the user on how they can solve that issue. Other than that, it's looks great to me :smiley:

Sorry for the back and forth :pray: It was not really clear to me what the issue was. I followed the task specification but that was not really what we had to do. For the next step, we will have to contact the editor team because we are updating their module.

Reply:
Also: Can you update the PR description and the commit message to be inline with the changes we made? Thanks in advanced.

Reply:
Done changes :+1: 

Thanks in Advance :blush: